lib/jiff-client.js

```diff
@@ -3752,6 +3790,40 @@
     jiff.helpers.Zp_equals = function (s1, s2) {
       return s1.Zp === s2.Zp;
     };
+    
+    /**
+     * Check that an integer is prime. Used to safely set the modulus Zp.
+     * @method is_prime
+     * @memberof jiff-instance.helpers
+     * @instance
+     * @param {number} p - the prime number candidate.
+     * @return {boolean} true if p is prime, false otherwise.
+     */
```

Comment:
This function is duplicated. I think I understand why you put it twice: you cannot use this inside make_jiff since jiff is being made at that point, and if you only have the function above, then it will not be exported to the developers to use on their own if they want to.

My suggestion is to remove this function definition here and use only the function definition above. But then, export it towards the end of the file as part of the exports.utils/jiff.utils object/namespace, similar to encrypt_and_sign and decrypt_and_sign.

If you do that, make sure you modify the JSDOC to have @memberof jiff.utils